src/pudl/transform/gridpathratoolkit.py

```diff
@@ -0,0 +1,161 @@
+"""Transformations of the GridPath RA Toolkit renewable generation profiles.
+
+Wind and solar profiles are extracted separately, but concatenated into a single table
+in this module, as they have exactly the same structure. The generator aggregation group
+association tables for various technology types are also concatenated together.
+
+Note that this transform is a bit unusual, in that it is producing a highly processed
+output table. That's because we're working backwards from an archived finished product
+to be able to provide a minimum viable product. If it goes well we will integrate or
+reimplement the steps leading up to this output table later.
+"""
+
+import pandas as pd
+from dagster import AssetCheckResult, AssetIn, asset, asset_check
+
+import pudl
+
+
+def _transform_capacity_factors(
+    capacity_factors: pd.DataFrame, utc_offset: pd.Timedelta
+) -> pd.DataFrame:
+    """Basic transformations that can be applied to many profiles.
+
+    - Construct a datetime column and adjust it to be in UTC.
+    - Reshape the table from wide to tidy format.
+    - Name columns appropriately.
+    """
+    # Convert timestamp to UTC before stacking!
+    # TODO: Ensure that we're using the same hour starting vs. ending convention
+    # here as we use in the FERC-714 and EPA CEMS data. Should this be reflected in
+    # the column names here and elsewhere, whatever the convention is?
```

Comment:
is this todo still relevant?

Reply:
Unfortunately it is still relevant but I punted on it here because it means digging into wtf is going on in our other hourly data which felt OOS for this PR & the GridLab work.

Reply:
okay... that's not idea but is definitely oos